synapse/config/logger.py

```diff
@@ -81,11 +81,18 @@
 """
 )
 
+NO_LOG_FILE_ERROR = """\
+Support for the log_file configuration option and --log-file commandline option was
```

Comment:
```suggestion
Support for the log_file configuration option and --log-file command line option was
```

Reply:
hrrmm maybe. I'll compromise on `command-line` :)

Reply:
Works for me :p

Reply:
Works for me :p